sql/5_0_0-to-5_0_1_upgrade.sql

```diff
@@ -129,6 +129,24 @@ VALUES ('page_validation','add_edit_event#theform_groups','/interface/main/calen
 INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `notes`, `activity`) VALUES ('page_validation', 'common#new-encounter-form', '/interface/forms/newGroupEncounter/common.php', 160, '{pc_catid:{exclusion: ["_blank"]}}', 1);
 #EndIf
 
+-- 1. users
+--
+#IfMissingColumn users weno_prov_id
+ALTER TABLE `users` ADD `weno_prov_id` VARCHAR(15) NULL AFTER `newcrop_user_role`;
+#Endif
+
+-- 2. prescriptions 
+--
+#IfMissingColumn prescriptions ntx 
+ALTER TABLE `prescriptions` ADD `ntx` INT(2) NULL DEFAULT NULL AFTER `prn`, ADD `rtx` INT(2) NULL DEFAULT NULL AFTER `ntx`, ADD `txDate` DATE NOT NULL AFTER `rtx`;
+#Endif
+
+-- 3. pharmacies 
+--
+#IfMissingColumn pharmacies ncpdp
+ALTER TABLE `pharmacies` ADD `ncpdp` INT(12) NULL AFTER `email`, ADD `npi` INT(12) NULL AFTER `ncpdp`;
+#Endif
+
 #IfNotTable therapy_groups
 CREATE TABLE `therapy_groups` (
   `group_id` int(11) NOT NULL auto_increment,
```

Comment:
sql changes need to go into following:
1. sql/database.sql (for new installations)
2. sql/5_0_0-to-5_0_1_upgrade.sql (for upgraders)

So, these sql changes also need to be in sql/database.sql